doc/obs_integration.md

```diff
@@ -19,7 +19,21 @@ The [systemsmanagement:Agama:Devel](https://build.opensuse.org/project/show/syst
 contains the latest released version of the Agama project. The packages should
 be more stable than in the Staging project.
 
-These packages are updated manually when a new version is released.
+These packages are updated automatically when a new version is released. See
+more details below.
+
+## Releasing a New Version
+
+For releasing a new version just create a new version tag in form `v[0-9]+\..*`
+and push it to the server:
+
+```
+git tag -a -m "Version 2.3.4" v2.3.4
```

Comment:
It would be even better if we sign the tag using the `-s` option. What do you think?

Reply:
OK, let's the one adding the tag decide.

Reply:
Added a note about this.